doc/lua_api.txt

```diff
@@ -2491,6 +2491,42 @@ For the following functions `x` can be either a vector or a number:
 
 Helper functions
 ----------------
+
+Note: calling functions and copying are fairly expensive in plain Lua, so weigh
+up the cost of performance in areas with lots of calls against maintenance and
+readability.
+
+### Table helpers
+
+* `table.copy(t)`: returns a table
+    * returns a deep copy of `table`
+* `table.count(t)`: number of keys in t.
+* `table.random(t)`: returns random element from `t`, an array.
+* `table.index_of(t, v)`: find key of v in t.
+* `table.map(t, f)`:
+    * `f` - function(element), return new value
+    * Returns a new table by iterating over all keys and running f to get value
+    * `t` is not modified.
+* `table.map_inplace(t, f)`:
+    * `f` - function(element), return new value
+    * Each element of `t` is passed to `f` and replaced with the returned value.
+    * `t` **is** modified.
+
+### String helpers
+
+* `string.split(str, separator, include_empty, max_splits, sep_is_pattern)`
+    * `separator`: string, default: `","`
+    * `include_empty`: boolean, default: `false`
+    * `max_splits`: number, if it's positive, splits aren't limited,
```

Comment:
This was only moved but the description seems to be wrong:
`if it's positive, splits are limited,`